lib/ansible/config/module_defaults.yml

```diff
@@ -158,6 +166,8 @@ groupings:
   - aws
   ec2_lc_find:
   - aws
+  ec2_metadata_facts:
```

Comment:
I think this was probably intentionally left out since the typical common AWS arg spec for credentials doesn't apply.

Reply:
removed